internal/logger/logger.go

```diff
@@ -48,14 +40,14 @@ func newLogger(levelEnabler zapcore.LevelEnabler, additionalCores ...zapcore.Cor
 	)
 	cores := append(additionalCores, defaultCore)
 
-	return &Logger{zap.New(zapcore.NewTee(cores...), zap.AddCaller()).Sugar()}
+	return zap.New(zapcore.NewTee(cores...), zap.AddCaller())
 }
 
 /*
 This function initialize klog which is used in k8s/go-client
 */
-func initKlog(log *Logger, level zapcore.Level) {
-	zaprLogger := zapr.NewLogger(log.WithContext().Desugar())
+func initKlog(logger *zap.Logger, level zapcore.Level) {
+	zaprLogger := zapr.NewLogger(logger)
```

Comment:
We weren't using the `SugaredLogger` before either, right? Since I see the call to the `.Desugar()` function.

Reply:
Was there any purpose in creating a Sugared one and "desugaring" it later on? Compared to the current approach of directly creating a normal one?